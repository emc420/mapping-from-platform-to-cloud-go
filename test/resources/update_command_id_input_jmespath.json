{
  "id": "newCommand",
  "input": {
    "value": "{{ @.input.prop2 }}"
  }
}