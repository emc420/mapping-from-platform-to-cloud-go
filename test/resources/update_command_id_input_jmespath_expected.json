{
  "id": "newCommand",
  "input": {
    "value": {
      "prop3": "value"
    }
  }
}